admin/class-yoast-notification-center.php

```diff
@@ -173,7 +180,11 @@ public static function maybe_dismiss_notification( Yoast_Notification $notificat
 			$is_dismissing = ( $notification_id === self::get_user_input( 'notification' ) );
 		}
 
-		// Can be dismissed by dismissal_key or notification_id.
+		// Fallback to ?dismissal_key=1&nonce=bla when JavaScript fails.
+		if ( ! $is_dismissing ) {
+			$is_dismissing = ( '1' === self::get_user_input( $dismissal_key ) );
+		}
+
 		if ( ! $is_dismissing ) {
```

Comment:
This part isn't necessary, because the same existence check is done above, you can return the value over there...